src/components/Resources/index.js

```diff
@@ -44,7 +44,19 @@ const useStyles = makeStyles(theme => ({
   },
   avatar: {
     backgroundColor: red[500]
-  }
+  },
+  textArea: {
+    fontFamily: '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
```

Comment:
would it be possible to use `fontFamily: inherit`?

Reply:
Of course, I'll update it and resend the PR

Reply:
Updated the fontFamily, let me know if anything else needs changing

Reply:
looks good to me,, but to be clear: I'm just a random person browsing hacktoberfest, not a maintainer of this project.

Reply:
Haha, thanks to you both, @Dan2D and @Aprillion